ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-internet-exit.j2

```diff
@@ -0,0 +1,51 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - router internet exit #}
+{% if router_internet_exit is arista.avd.defined %}
+
+### Router Internet Exit
+
+{%     if router_internet_exit.exit_groups is arista.avd.defined %}
+#### Exit Groups Summary
+
+| Exit Groups | Local Connections | Fib Default |
+| ----------- | ----------------- | ----------- |
+{%         for exit_group in router_internet_exit.exit_groups %}
+{%             for local_connection in exit_group.local_connections | arista.avd.default([]) %}
+| {{ exit_group.name }} | {{ local_connection.name }} | - |
+{%             endfor %}
+{%             if exit_group.fib_default is arista.avd.defined(true) %}
+| {{ exit_group.name }} | - | True |
+{%             endif %}
```

Comment:
as commented above - for one exit-group there should be one row - use `join("<br>")` as in

https://github.com/aristanetworks/avd/blob/57b85d5868ab0819634ac09e09fd51d8abd126f9/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dps-interfaces.j2#L36

Reply:
Done!